CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+## 1.1.1 (2021-09-24)
+- Remove current cc implementation
```

Comment:
rather than remove current implementation (i think ambiguous) maybe these 2 messages can be combine into `Change cc implementation using Invoice flow` or similar

Reply:
In addition to kevin's comment, we should also update the version to 1.2.0 since this is not a patch / bug fix.